adapters/bombcrypto-bnb/index.js

```diff
@@ -0,0 +1,145 @@
+require("dotenv").config();
+
+const moment = require("moment");
+const Binance = require("../../sdk/binance");
+const axios = require("axios");
+const URL = "http://nft-sales-service.dappradar.com/open-source";
+const KEY = process.env.DAPPRADAR_API_KEY;
+const path = require("path");
+const BigNumber = require("bignumber.js");
+
+class BombCrypto {
+    constructor() {
+        this.name = "bombcrypto-bnb";
+        this.symbol = "BNB";
+        this.token = "bnb";
+        this.protocol = "binance";
```

Comment:
We don't support this value as a `protocol`. Change it to `bsc`, please.

Reply:
Yeah missed that, fixed it now